### ISSUE TITLE
SuperPutty opens session in a new Putty terminal window

### ISSUE THREAD

**Issue by schlos** [jimradford/superputty#583]
What steps will reproduce the problem?
1. Open SuperPuTTY
2. Connect to a session
3. Putty opens in another Putty window

What is the expected output?
SSH terminal session should open as indiviual tab within SuperPuTTY.

What do you see instead?
New Putty window was opened.

What version of the product are you using? On what operating system?
SuperPuTTY 1.4.0.8
Putty 0.63
Windows 8.1 x64 with all the latest updates

Some of my sessions use KerberOS.

Related to Legacy issues:
Putty sessions not embedded in SuperPuTTY #307
SuperPutty opens sessions in a new Putty Window #92

**Comment by BogdanelBM** [comment]
same problem.. have you fixed it so far?

**Comment by w-** [comment]
super old bug that apparently is still happening.

SuperPuTTY 1.4.0.9
putty 0.70
windows 10 pro x64

installed super putty and putty using choclatey in case that matters.

[SuperPuTTY.20190303_160601.log](https://github.com/jimradford/superputty/files/2922921/SuperPuTTY.20190303_160601.log)

**Comment by uberturkey** [comment]
Still happening.... Anyone know how to fix?
SuperPuTTy 1.4.0.8
putty 0.76
Windows 10

**Comment by jimradford** [comment]
@uberturkey Have you tried with the latest 1.4.0.10 or the development (1.4.0.11) versions of SuperPuTTY?

**Comment by uberturkey** [comment]
I have not, I am unable to due to a locked down PC.  Is there a setting or fix to this issue?  Thank you for the reply.

**Comment by jimradford** [comment]
@uberturkey No, this issue has cropped up from time to time, the only concrete cause I've seen was caused by TeamViewer. It is possible other programs might cause this issue as well.

I'm not going to be able to assist you with this issue if you are not running the latest release version of SuperPuTTY. Have you considered speaking to your system administrators to see if you can get the latest version installed, or restrictions lifted so you can better help me to help you?

**Comment by gigapup** [comment]
Hi jimradford, I'm facing the same issues and after reading this post I've downloaded the 1.4.0.10 but the issue still happens Do you have any news about this?

**Comment by Jeklah** [comment]
I'm also encountering this issue, I have the latest version of super putty.
